tests/foreman/virtwho/cli/test_esx.py

```diff
@@ -34,9 +35,12 @@
 from robottelo.virtwho_utils import get_configure_file
 from robottelo.virtwho_utils import get_configure_option
 from robottelo.virtwho_utils import hypervisor_json_create
+from robottelo.virtwho_utils import hypervisorsettings
 from robottelo.virtwho_utils import virtwho_package_locked
 from robottelo.virtwho_utils import VIRTWHO_SYSCONFIG
 
+HYPERVISOR_TYPE = 'esx'
```

Comment:
hypervisor_type is defined by hypervisorsettings.esx.hypervisor_type in form_data

Reply:
Use the form_date['hypervisor_type'] instead of it.